src/scripts/main.js

```diff
@@ -1 +1,60 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+    document.addEventlistener('click', () => {
+        resolve('First promise was resolved');
+    });
+
+    setTimeout(() => {
+        const error = new Error('First promise was rejected');
+
+        reject(error);
+    }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+    document.addEventListener('click', (event) => {
```

Comment:
```suggestion
    document.addEventListener('click', (ev) => {
```
![image](https://user-images.githubusercontent.com/96409563/216271163-b4b8af3d-a17b-4a54-85d9-4f7d6cf2091b.png)